src/util/prime.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2020 Chaintope Inc.
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+//! Utility for prime number
+//!
+//!
+
+use rug::Integer;
+use rug::integer::Order;
+
+/// Prime number for secp256k1 field element.
+pub const P: [u8; 32] = [
+    0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
+    0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
+    0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
+    0xff, 0xff, 0xff, 0xfe, 0xff, 0xff, 0xfc, 0x2f
+];
+
```

Comment:
This field element should be managed by rust-secp256k1?

Reply:
Agree, [this file](https://github.com/rust-bitcoin/rust-secp256k1/blob/master/src/constants.rs) can be hold this value. 
I'm going to try to create PR.

Reply:
OK. For now, merge this, but replace when the pull request is merged.